ci/kind/kind-setup.sh

```diff
@@ -152,10 +153,49 @@ function configure_networks {
   # create new bridge network per subnet
   i=0
   networks=()
-  for s in $SUBNETS; do
+  bridge_id=$(docker network inspect kind -f {{.ID}})
```

Comment:
how do the changes in this function relate to the `docker_run_with_host_net iptables -I DOCKER-USER -j ACCEPT` command that we already execute?

Reply:
You just reminded me. I didn't notice this. This command is sufficient to allow connections across bridges. I have removed the changes.